docs/3.0.0-beta.x/migration-guide/migration-guide-alpha.26-to-beta.md

```diff
@@ -0,0 +1,736 @@
+# Migration Guide
+
+Upgrading your Strapi application to `v3.0.0-beta.x`.
+
+To upgrade a project to the `beta` version of Strapi follow the instructions below.
+
+## Updating global Strapi installation
+
+If you have installed Strapi globally. You will need to upgrade its version:
+
+```bash
+npm install -g strapi@beta
+# or
+yarn global add strapi@beta
+```
+
+## Updating dependencies
+
+First let's clean your project's dependencies and update its package.json file.
+
+### Clean your `node_modules`
+
+```bash
+rm -rf node_modules
+```
+
+### Update your `package.json`
+
+Start by upgrading all your strapi dependencies to the current `beta`.
+
+To find the current `beta` version to use in `package.json` run:
+
+```bash
+npm view strapi@beta version
+# 3.0.0-beta.x
+```
+
+#### `Dependencies`
+
+You must upgrade the versions and add `strapi-admin` as a dependency.
+
+**Before**
+
+```json
+{
+  "dependencies": {
+    //...
+    "strapi": "3.0.0-alpha.26.2",
+    "strapi-hook-bookshelf": "3.0.0-alpha.26.2",
+    "strapi-hook-knex": "3.0.0-alpha.26.2",
+    "strapi-utils": "3.0.0-alpha.26.2"
+  }
+}
+```
+
+**After**
+
+```json
+{
+  "dependencies": {
+    //...
+    "strapi": "3.0.0-beta.x",
+    "strapi-admin": "3.0.0-beta.x",
+    "strapi-hook-bookshelf": "3.0.0-beta.x",
+    "strapi-hook-knex": "3.0.0-beta.x",
+    "strapi-utils": "3.0.0-beta.x"
+  }
+}
+```
+
+Starting from this new release plugins are now npm dependencies so, you need to install them too.
+
+Here are the default ones. If you have installed other plugins you will also need to add them here (e.g `strapi-plugin-graphql`).
+
+```json
+{
+  "dependencies": {
+    //...
+    "strapi-plugin-content-manager": "3.0.0-beta.x",
+    "strapi-plugin-content-type-builder": "3.0.0-beta.x",
+    "strapi-plugin-email": "3.0.0-beta.x",
+    "strapi-plugin-settings-manager": "3.0.0-beta.x",
+    "strapi-plugin-upload": "3.0.0-beta.x",
+    "strapi-plugin-users-permissions": "3.0.0-beta.x"
+  }
+}
+```
+
+#### `Scripts`
+
+We have completely refreshed the `scripts` of a project so update them as follows:
+
+**Before**
+
+```json
+{
+  "scripts": {
+    "setup": "cd admin && npm run setup",
+    "start": "node server.js",
+    "strapi": "node_modules/strapi/bin/strapi.js",
+    "postinstall": "node node_modules/strapi/lib/utils/post-install.js"
+    //...
+  }
+}
+```
+
+**After**
+
+```json
+{
+  "scripts": {
+    "develop": "strapi develop",
+    "start": "strapi start",
+    "build": "strapi build",
+    "strapi": "strapi"
+    //...
+  }
+}
+```
+
+::: warning
+
+To avoid confusions we have decided to give the same behaviour to the `npm run start` and `strapi start` commands. To do so we had to introduce a new `strapi develop` command to run your project in `watch` mode.
```

Comment:
To avoid confusions we have decided to give the same **behavior** to the `npm run start` and `strapi start` commands. To do so we had to introduce a new `strapi develop` command to run your project in `watch` mode.